modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+* [ ] <<continuous-db>>
+* [ ] <<debug-bundle>>
+* [ ] <<topic-rf>>
+* [ ] <<maintenance-mode>>
+* [ ] <<decom-state>>
+
+
+
+=== Level 2 checklist
+
+[%interactive]
+* [ ] Dev / test environment available?
+* [ ] Metrics being captured and stored
+* [ ] System logs being captured and stored
+* [ ] Upgrade policy defined and implemented
+* [ ] Rack awareness configured correctly
+
+
+=== Level 3 checklist
+
+[%interactive]
+* [ ] Alerting configured for metrics
+* [ ] Backup or DR solution in place
+* [ ] Backup or DR solution tested
+* [ ] DR: is replication in place and is replication monitored
+* [ ] Is deployment automation used and could it be used to rebuild the cluster if required
+* [ ] Are audit logs being forwarded to an enterprise SIEM system
+
+
+== Level 1 Production Readiness
+
+[[redpanda-license]]
+=== Load Redpanda License
+
+Ensure that the Redpanda License has been loaded into the cluster configuration. This is required to enable Enterprise features described later. 
+
+Example:
+
+```bash
+rpk cluster license info
+```
+
+Output:
+
+```bash
+LICENSE INFORMATION
+===================
+Organization:      Redpanda Owlshop LLC
+Type:              enterprise
+Expires:           Mar 25 2025
+```
+
+[[cluster-health]]
+=== Check your cluster's health
+
+Establish that all brokers are connected and running. Use rpk to check the health of the cluster. No nodes should be down. There should be zero leaderless partitions or under-replicated partitions.
+
+The cluster should be listed as healthy.
+
+Example:
+
+```bash
+rpk cluster info
+```
+
+Output:
+
+```bash
+CLUSTER
+=======
+redpanda.be267958-279d-49cd-ae86-98fc7ed2de48
+
+BROKERS
+=======
+ID    HOST            PORT  RACK
+0*    54.70.51.189    9092  us-west-2a
+1     35.93.178.18    9092  us-west-2b
+2     35.91.121.126   9092  us-west-2c
+```
+
+Example:
+
+```bash
+rpk cluster health
+```
+
+Output:
+
+```bash
+CLUSTER HEALTH OVERVIEW
+=======================
+Healthy:                          true
+Unhealthy reasons:                []
+Controller ID:                    0
+All nodes:                        [0 1 2]
+Nodes down:                       []
+Leaderless partitions (0):        []
+Under-replicated partitions (0):  []
+```
+
+[[production-mode]]
+=== Production mode enabled
+
+Ensure that Redpanda is running in production mode. Production mode is set via the cluster configuration and can be checked on any node via `rpk`. If either configuration is set to `true`, the cluster is not in full production mode and must be corrected.
+
+Example:
+```bash
+grep -E 'developer_mode|overprovisioned' /etc/redpanda/redpanda.yaml
+    developer_mode: false
+    overprovisioned: false
+```
+Both `developer_mode` and `overprovisioned` should be `false` (or not present).
+
+[[redpanda-reqs]]
+=== Check if the system meets Redpanda requirements
+
+Run a system check to ensure that your system meets Redpanda's requirements.
+
+Example:
+
+```bash
+sudo rpk redpanda check
+```
+
+Output:
```

Comment:
```suggestion
.`Output`
```

Reply:
So...what specific result in this output must be met/what should it say? Is there any item that can be false and your system still meets RP requirements?